internal/bminventory/inventory.go

```diff
@@ -586,6 +589,15 @@ func (b *bareMetalInventory) RegisterClusterInternal(
 	}
 
 	if swag.StringValue(params.NewClusterParams.HighAvailabilityMode) == models.ClusterHighAvailabilityModeNone {
+		// verify minimal OCP version
+		ocp_version, err := version.NewVersion(swag.StringValue(params.NewClusterParams.OpenshiftVersion))
+		minimal_version_for_sno, err := version.NewVersion(minimalOpenShiftVersionForSingleNode)
+		if ocp_version.LessThan(minimal_version_for_sno) {
+			err := errors.Errorf("Invalid OCP version (%s) for HighAvailabilityMode=%s",
```

Comment:
I think we should change the message to: "Invalid OCP version for Single node, it's supported for version 4.8 & up".
Not sure that the user knows what it means HighAvailabilityMode=None
@avishayt WDYT?